util/mod_translation_updater.py

```diff
@@ -121,9 +120,9 @@ def main():
 
 # Group 2 will be the string, groups 1 and 3 will be the delimiters (" or ')
 # See https://stackoverflow.com/questions/46967465/regex-match-text-in-either-single-or-double-quote
-pattern_lua_quoted = re.compile(r'[\.=^\t,{\(\s]N?F?S\s*\(\s*(["\'])((?:\\\1|(?:(?!\1)).)*)(\1)[\s,\)]', re.DOTALL)
+pattern_lua_quoted = re.compile(r'(?:^|[\.=^\t,{\(\s])N?F?S\s*\(\s*(["\'])((?:\\\1|(?:(?!\1)).)*)(\1)[\s,\)]', re.DOTALL)
```

Comment:
second suggestion: annotate the regex so one doesn't need a perl diploma to read it
(I think this works)
```python
pattern_lua_quoted = re.compile(
    r'(?:^|[\.=^\t,{\(\s])' # look for beginning of line or anything that isn't an identifier
    r'N?F?S\s*\(\s*' # match an NS FS or S function call
    r'(["\'])((?:\\\1|(?:(?!\1)).)*)(\1)' # quoted string
    r'[\s,\)]', # end of call or argument
    re.DOTALL
)
```